spec/models/user_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe "validations" do
+    USER_FIELDS = %i(email country).freeze
+    subject { FactoryGirl.build(:user) }
+
+    USER_FIELDS.each do |attr|
+      it { is_expected.to validate_presence_of attr }
+    end
+
+    it { is_expected.to validate_length_of :email }
+  end
+
+  context "active user" do
+    it "#active_for_authentication?" do
+      expect(subject.active_for_authentication?).to be true
+    end
+
+    it "#inactive message?" do
+      expect(subject.inactive_message).to eq :inactive
+    end
+
+    it "#soft_delete" do
+      @current_time = Time.now
+      allow(Time).to receive(:current).and_return(@current_time)
```

Comment:
It's common practice to use the [Timecop](https://github.com/travisjeffery/timecop) gem to manage time-based tests. I'd suggest installing it and using it in place of stubbing here as it'll be something we'd want to use going forward so you can help set the precedent :)